data-pipeline/src/trace_exporter.rs

```diff
@@ -99,59 +97,59 @@ fn add_path(url: &Uri, path: &str) -> Uri {
     Uri::from_parts(parts).unwrap()
 }
 
-struct DroppedP0Counts {
-    pub dropped_p0_traces: usize,
-    pub dropped_p0_spans: usize,
-}
+// struct DroppedP0Counts {
```

Comment:
I would recommend for any blocks of code that we're temporarily commenting out we have a `// TODO: APMSP-XXXX re-enable CSS` comment, or something along those lines to make sure we don't miss anything when we add back the CSS functionality. 

Also you can use `/*` and `*/` for block comments instead of each line being a `//`. It's a little bit cleaner.

Reply:
Ahh yeah, I'm just so used to the rust rover cmd+/ to just line comment whole sections :P